### PR TITLE
Contributor Comms reorganization

### PR DESCRIPTION
**Pull request by mbbroberg**
Significant clean up effort for Upstream Marketing documentation
focused on:

- Ensure roles described how the team has functioned over the last year
- Fill in stale TODOs with solid information
- Organize resources into the storytelling folder

Signed-off-by: Matthew Broberg <matt@opensource.com>

### CODE REVIEW

**Comment by mbbroberg** [comment]
Hey @parispittman and @mrbobbytables, I did some work to tidy up based on a few discussions of @kaslin @rajula96reddy @chris-short and myself. Please take a look as time permits and happy to chat or async about the specifics.

**Comment by parispittman** [comment]
communication/marketing-team/role-handbooks/council.md:9:115: "Coucil" is a misspelling of "Council"

looks great!! I think having each role get defined terms would be the next iteration. Many of the named roles outside of reviewer and approver have terms on the project and that helps escalate the shadow program behind it as well. Examples: two years for steering committee, release team is 3 months per role but building up to lead which will be ~9 months, event team is per event, etc.

**Comment by parispittman** [comment]
/hold for steering lgtms 

steering - there is minor change here to the new example roles section in sig-governance.md

**Comment by mbbroberg** [comment]
Thanks for the review @parispittman! I fixed the typo and reformatted the table to show available shadow opportunities by roles in practice of Program Lead, Promotion Lead, and Liaison Lead. I hope that helps. 

Note we have a lot of interest in shadow opportunities that we're hoping to review the week after KubeCon. I'm hopeful we can get this merged by that timeline if that could work.

**Comment by mbbroberg** [comment]
Oops, seems I left this incomplete. Going to rerun tests now: 

/retest-required

**Comment by justaugustus** [comment]
> Oops, seems I left this incomplete. Going to rerun tests now:
> 
> /retest-required

@mbbroberg -- Ping on the test fixes :)

```console
Verifying verify-spelling.sh
communication/marketing-team/README.md:13:120: "avaiable" is a misspelling of "available"
exit status 2
FAILED   verify-spelling.sh	2s 
```

ref: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/community/5989/pull-community-verify/1456000700650622976

**Comment by mbbroberg** [comment]
@justaugustus you're a gentleman and a scholar 🙇

File: committee-steering/governance/sig-governance.md
Comment (justaugustus) [inline committee-steering/governance/sig-governance.md:276]:
```suggestion
[Marketing Council]: /communication/marketing-team/role-handbooks/council.md
```

Style nit per https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#links.

**Review by justaugustus — CHANGES_REQUESTED** [review]
Itty bitty style nit in `committee-steering/governance/sig-governance.md`.

Since you're in the neighborhood, it would be super appreciated if you'd be up for conforming the other links at the bottom of that doc to the style guide (in a separate commit).

No biggie, if not! :)

**Comment by mbbroberg** [comment]
@justaugustus thank you thank you for the nit / nudges 🙏 -- it's always nice to be helpful when I'm in the neighborhood. I'll update that and then, @parispittman, how are we looking on the front of removing the hold?

**Comment by chris-short** [comment]
This is an amazing body of work @mbbroberg. Thank you.

**Review by justaugustus — APPROVED** [review]
> @justaugustus thank you thank you for the nit / nudges 🙏 -- it's always nice to be helpful when I'm in the neighborhood. I'll update that and then, @parispittman, how are we looking on the front of removing the hold?

@mbbroberg -- Thanks so much!
Changes to `committee-steering/governance/sig-governance.md` look great, so approving on behalf of Steering.
/approve

Feel free to lift the hold once you get an /lgtm + /approve from @kubernetes/sig-contributor-experience / Marketing folks!

**Comment by justaugustus** [comment]
> Feel free to lift the hold once you get an /lgtm + /approve from @kubernetes/sig-contributor-experience / Marketing folks!

/assign @parispittman @rajula96reddy @kaslin

**Pull request merged**
